docs/docker.md

```diff
@@ -18,7 +18,7 @@ Define your specific settings in **secmonkey.env** file. For example, this file
     SESSION_COOKIE_SECURE=False
 
 
-**Please note** to be able to run the scheduler inheriting the SecurityMonkeyInstanceProfile IAM Role, the AWS credentials have to be removed from config file above (secmonkey.env) otherwise boto3 inside the scheduler container won't escalate to SecurityMonkeyInstanceProfile because will try to use empty AWS credentials instead.
+**Please note** to be able to run the scheduler container properly inheriting the SecurityMonkeyInstanceProfile IAM Role, AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY have to be completely removed (or commented) from secmonkey.env, otherwise boto3 inside the scheduler container won't escalate to SecurityMonkeyInstanceProfile because will try to use empty AWS credentials.
```

Comment:
The final part of the sentence is missing the word it:
> because **it** will try to use empty AWS credentials.

Reply:
@fabiop : please make the requested change and I will merge this in.

Reply:
@mikegrima ok, change done, thank you